pkg/install/templates/flux-config.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+{{ if .ConfigFileContent -}}
```

Comment:
Sorry I am noticing this now. But we should probably handle the Memcached deployment in the same way with `.RegistryDisableScanning` (right now we exclude it in the installation code).